config/initializers/cors.rb

```
@@ -5,12 +5,21 @@
-# end
+#      resource "*",
+#        headers: :any,
+#        methods: [:get, :post, :put, :patch, :delete, :options, :head]
+#    end
+  else Rails.env.development?
+    allow do
+      origins "*"
+      resource "*",
+        headers: :any,
+        methods: [:get, :post, :put, :patch, :delete, :options, :head]
+    end
```

rubocop (1.69.1)

[convention] Inconsistent indentation detected.

(Layout/IndentationConsistency)